cmd/fleet/serve_test.go

```diff
@@ -91,6 +91,12 @@ func TestMaybeSendStatistics(t *testing.T) {
 			LicenseTier:                          "premium",
 			NumHostsEnrolled:                     999,
 			NumUsers:                             99,
+			NumSoftwares:                         100,
+			NumHostSoftwares:                     101,
```

Comment:
```suggestion
			NumHostSoftwareVersions:                     101,
```

Small nit. Suggest being consistent w/ "software versions" naming we use in the API: https://fleetdm.com/docs/rest-api/rest-api#list-software-versions

Also what's the difference between `NumSoftware` and `NumHostSoftware`?